server/src/test/java/org/apache/druid/server/coordinator/loading/HttpLoadQueuePeonTest.java

```diff
@@ -76,6 +82,15 @@ public class HttpLoadQueuePeonTest
   public void setUp()
   {
     httpClient = new TestHttpClient();
+    JacksonConfigManager configManager = EasyMock.createNiceMock(JacksonConfigManager.class);
+    EasyMock.expect(
+        configManager.watch(
+            EasyMock.eq(CoordinatorDynamicConfig.CONFIG_KEY),
+            EasyMock.anyObject(Class.class),
+            EasyMock.anyObject()
+        )
+    ).andReturn(new AtomicReference<>(CoordinatorDynamicConfig.builder().build())).anyTimes();
+    EasyMock.replay(configManager);
```

Comment:
I don't think `configManager` is being used anywhere. We can remove this.

Reply:
Thanks for catching this! It was initially needed, but not needed after switching to a supplier, missed this!